code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -487,7 +487,7 @@
 
 /obj/item/modular_computer/screwdriver_act(mob/user, obj/item/tool)
 	if(!length(all_components))
-		to_chat(user, span_warning("This device doesn't have any components installed."))
+		user.balloon_alert(user, "This device doesn't have any components installed.")
```

Comment:
balloon alerts should be short and also without capitalization and punctuation

Reply:
Thank you! Making edits now.